lib/honey_format/matrix/row_builder.rb

```diff
@@ -57,14 +57,26 @@ def build_row!(row)
 
       # Convert values
       @type_map.each do |column, type|
-        row[column] = @converter.call(row[column], type)
+        if row.respond_to?(column) && wrap_in_array(@columns).include?(column)
```

Comment:
@buren Hey, I don't know if this case really exists: `builds struct from single symbol`.
https://github.com/buren/honey_format/blob/master/spec/honey_format/matrix/row_builder_spec.rb#L29

The method `wrap_in_array` was implemented exclusively to deal with this case present on the spec mentioned.

If it does not exist I can remove it, the performance will be better.